### PR TITLE
Stanford submission

### PR DESCRIPTION
**Pull request by gtklocker**
Will be updating the following statement as the PR progresses:
We are now at 14 pages.

### CODE REVIEW

**Comment by gtklocker** [comment]
We should probably not merge this btw.

**Comment by gtklocker** [comment]
@dionyziz we're currently sitting at 14 pages. If I move consumption to the appendix we'll make it 11, should I do it? Or trim something else?

**Comment by dionyziz** [comment]
Yes, move consumption to appendix. Please also keep this as a separate branch.

File: abstract.tex [hunk 1]
```diff
@@ -16,19 +16,4 @@
   Next, we design a novel construction for burning which is simple
   and flexible, making it compatible with all existing popular cryptocurrencies.
   We prove our scheme is secure in the Random Oracle model.
-  We explore the application of destroying
-  value in a legacy cryptocurrency to bootstrap a new one.
-  The user burns coins in the source blockchain
-  and subsequently creates a proof-of-burn, a
-  short string proving that the burn took place, which she then submits to
```
Comment (dionyziz) [inline abstract.tex]: We should still mention that proofs-of-burn are possible.

**Review by dionyziz — CHANGES_REQUESTED** [review]
Probably need to amend our list of contributions also.

**Comment by dionyziz** [comment]
Please anonymize for submission.

**Comment by gtklocker** [comment]
It says it's optional to anonymise:

> Papers can optionally be anonymized. The authors can decide to optionally anonymize the submission by omitting the author names in the pdf. Author names will be recorded in either case in HotCRP but not displayed to the reviewers.

Do we have any good reason to do it?

File: analysis.tex [hunk 2]
```diff
@@ -93,13 +93,100 @@ \section{Analysis}\label{sec:analysis}
 definition of computational indistinguishability is included in
```
Comment (dionyziz) [inline analysis.tex:92]: The complete proofs are not in the Appendix any more "for the complete proofs see Appendix".

**Review by dionyziz — CHANGES_REQUESTED** [review]

File: analysis.tex (hunk 2 above)
Comment (gtklocker) [reply analysis.tex:92]: fixed

**Pull request closed without merging**